### ISSUE TITLE
CUDA error: run out of memery when training

### ISSUE THREAD

**Issue by liujiahui802** [kazuto1011/deeplab-pytorch#28]
Hi , I'm trying to use your code on voc2012 dataset , when I train the net , it says CUDA error : out ou memory. I got 2 GPUs  and their information presented below: 

![image](https://user-images.githubusercontent.com/42512669/44437150-3d59e800-a5eb-11e8-9a2d-55e790c50503.png)

and error message is like this:
![image](https://user-images.githubusercontent.com/42512669/44437183-6da18680-a5eb-11e8-84b1-87a5465c749d.png)

May I ask how much memory did you command when training on coco stuff dataset on each GPU? And how can I slove this issue? Change GPU server or maybe somthing else?

**Comment by kazuto1011** [comment]
As I wrote [here](https://github.com/kazuto1011/deeplab-pytorch#training), memory usage was 11.2 GB on a single Titan X. Did you set ```CUDA_VISIBLE_DEVICES``` for your 2 GPUs appropriately?

**Comment by kazuto1011** [comment]
TLDR: ```CUDA_VISIBLE_DEVICES=0,1 python train.py --config ......```
